tests/visitors/test_models.py

```diff
@@ -1,9 +1,221 @@
 from parameterized import parameterized
 
-from django_codemod.visitors.models import ModelsPermalinkTransformer
+from django_codemod.visitors.models import (
+    ModelsPermalinkTransformer,
+    OnDeleteTransformer,
+)
 from tests.visitors.base import BaseVisitorTest
 
 
+class TestOnDeleteTransformer(BaseVisitorTest):
+
+    transformer = OnDeleteTransformer
+
+    def test_foreign_key(self) -> None:
+        pass
+
+    def test_one_to_one_field(self) -> None:
+        before = """
+            from django.db import models
+
+            class MyModel(models.Model):
+                migrations.CreateModel(
+                    fields=[
+                        (
+                            'model',
+                            models.OneToOneField(
+                                to = 'random.Model'),
+                        ),
+                    ],
+                )
+        """
+        after = """
+            from django.db import models
+
+            class MyModel(models.Model):
+                migrations.CreateModel(
+                    fields=[
+                        (
+                            'model',
+                            models.OneToOneField(
+                                to = 'random.Model', on_delete = models.CASCADE),
+                        ),
+                    ],
+                )
+        """
+        self.assertCodemod(before, after)
+
+    # testing our on_delete detector works as expected in
+    # all the little edge cases
```

Comment:
Can we remove this comment now please? I think it's clear that we're testing for edge cases.

If you feel like some tests deserve some explanations, I would prefer a docstring over a comment. That's not mandatory, though, usually a good function name is enough IMO.